docs/docs/mdx/markdown-syntax.md

```diff
@@ -34,11 +34,11 @@ Here's how those tags render in HTML:
 
 ###### heading 6
 
-- each heading gets converted to their HTML equivalent
+- Each heading gets converted to their HTML equivalent
```

Comment:
@mhkasif Please don't do such type of changes for sake of a PR

Reply:
none of your business